recipes/spark_historyserver.rb

```diff
@@ -34,11 +34,11 @@
   if node['spark']['spark_defaults'].key?('spark.eventLog.dir')
     node['spark']['spark_defaults']['spark.eventLog.dir']
   else
-    '/user/spark/applicationHistory'
+    'hdfs:///user/spark/applicationHistory'
```

Comment:
I just meant use the string formatting instead of prefixing inline 'hdfs://'.
Is the value of `dfs` not always "hdfs://"?